README.md

```diff
@@ -6,28 +6,88 @@ See our [CONTRIBUTING.md](CONTRIBUTING.md) for Git configuration and working wit
+
+## Working with the codebase
 
 There is some custom functionality apart from what's in the base Lando installation.
-* Installing a Drupal site from config (no DB needed): `lando si`
-* If switching to a new branch, always: `lando rebuild -y && lando si` to start off completely fresh.
+
 * See a list of shortcuts (Drush, Composer, etc.): `./robo.sh common:shortcuts-help`
-* Use `./composer.sh` instead of `composer` or `lando composer`. `./composer.sh` will cause entries `composer.log` to be made so we can replay composer commands on conflicts.
-* Sign in as admin: `./drush.sh uli`.
-* Export content as configuration `./robo.sh drupal-project:export-content`. See [Exporting Content as Configuration](#exporting-content-as-configuration).
+* When switching to a new branch, always: `lando rebuild -y && lando si` to start off completely fresh.
+* Export content as configuration `./robo.sh drupal-project:export-content`.
+  See [Exporting Content as Configuration](docs/backend#exporting-content-as-configuration).
+
+## Compiling theme assets
+
+See our [Frontend documentation](docs/frontend.md) for working with the `digital_gov` custom theme.
+
+## Basic command reference
+
+### Composer commands
+Instead of using `lando composer` we use `./composer.sh` which generate entries in `composer.log` so we can replay composer commands on conflicts.
+
+| **Command**                                  | **Use case**                |
+|----------------------------------------------|-----------------------------|
+| `./composer.sh require drupal/<MODULE_NAME>` | Download a drupal module    |
+| `./composer.sh remove drupal/<MODULE_NAME>`  | Remove a drupal module      |
+| `./composer.sh update --lock`                | Import Drupal configuration |
+
+### Drush commands
+
+| **Command**       | **Use case**                 |
+|-------------------|------------------------------|
+| `lando drush cr`  | Clearing Drupal cache        |
+| `lando drush uli` | Log into Drupal as superuser |
+| `lando drush cim` | Import Drupal configuration  |
+| `lando drush cex` | Export Drupal configuration  |
+
+### Lando commands
+
+| **Command**     | **Use case**                                         |
+|-----------------|------------------------------------------------------|
+| `lando start`   | Start the container                                  |
+| `lando stop`    | Stop the container                                   |
+| `lando rebuild` | Rebuild the container (retains your db)              |
+| `lando destroy` | Destroys container and your db (when all else fails) |
+
+### Custom Lando commands
+
+| **Command**            | **Use case**                                    |
+|------------------------|-------------------------------------------------|
+| `lando si`             | Install a fresh Drupal site from configuration. |
+| `lando export-content` | Export content as configuration.                |
```

Comment:
This all looks amazing Ray, I love all the extra help it provides.
The only thing I have is that, `lando export-content`, I actually fulfilled the @todo for that, but I did not remove it. That can be removed from `.lando.dist.yml`. The new command is `./robo.sh drupal-project:export-content`, although, that's not a lando command.
There are other custom lando commands if you want to add any here, like:
* `lando su`: Run updates, import configuration, run cron, etc (Install if not installed).
* `lando xdebug-on`: Enable Xdebug.
* `lando xdebug-off`: Disable Xdebug.
* `lando patch`: Apply composer patches or regenerate lock hash.
* `lando be`:  Builds backend (composer) dependencies.
* `lando fe`: Builds front end site (dependencies & compilation)

Reply:
Revisions are ready for review.